docs/mutations.md

```diff
@@ -85,7 +85,7 @@ def resolve_logout(_, info):
 
 > **Binding Mutation Resolvers**
 >
-> Recall that resolvers need to be bound to their respective resolvers via the `make_executable_schema` call. If you're following along from the introduction that call will look similar to the following:
+> Recall that resolvers need to be bound to their schema via the `make_executable_schema` call. If you're following along from the introduction that call will look similar to the following:
```

Comment:
```suggestion
> Recall that resolvers need to be bound to their fields in the schema via the `make_executable_schema` call. If you're following along from the introduction that call will look similar to the following:
```

Reply:
linking a resolver to a field name is achieved with a decorator `@mutation.field("exampleField")` (or `mutation.set_field(...)`)

linking the whole set of resolvers (defined in an instance of `MutationType`) to the schema is achieved with `make_executable_schema`

Reply:
No. `@obj.field` only tells `obj` which function should it use as resolver for field. It's `make_executable_schema` that actually puts python functions with fields in schema.